oadr2/base.py

```diff
@@ -3,7 +3,7 @@
 __author__ = 'Benjamin N. Summerton <bsummerton@enernoc.com>'
 
 import logging, threading
-import event, control
+from . import event, control
```

Comment:
I think it would be better to use absolute imports (in the other files too) than relative ones - it improves readability from where we import things.